src/main/java/org/symphonyoss/symphony/messageml/elements/TextArea.java

```diff
@@ -53,6 +54,8 @@ protected void buildAttribute(MessageMLParser parser,
       case NAME_ATTR:
       case REQUIRED_ATTR:
       case PLACEHOLDER_ATTR:
+      case MINLENGTH_ATTR:
```

Comment:
modify also Element#buildAttribute to manage these field if MinMaxLengthElement is implemented.
It will be not used now (as both TextArea and Textfield override this method) but just in case, for the future...